SDAPackage/SDA/stationary_obstacle.py

```diff
@@ -31,3 +31,22 @@ def get_height(self):
         The height of the obstacle
         """
         return self.height
+
+    def make_avoidance_points(self, drone_altitude):
+        """
+
+        :param drone_altitude: The altitude of the drone
+        :type drone_altitude: Float
+        """
+        new_avoidance_pos_points = [
+            [self.get_point()[0] + self.get_radius(), self.get_point()[1] + self.get_radius(), drone_altitude],
+            [self.get_point()[0] - self.get_radius(), self.get_point()[1] - self.get_radius(), drone_altitude],
+            [self.get_point()[0] + self.get_radius(), self.get_point()[1] - self.get_radius(), drone_altitude],
+            [self.get_point()[0] - self.get_radius(), self.get_point()[1] + self.get_radius(), drone_altitude],
+            [self.get_point()[0], self.get_point()[1] + self.get_radius(), self.get_height() + (Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS * 2)],
+            [self.get_point()[0], self.get_point()[1] - self.get_radius(), self.get_height() + (Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS * 2)],
+            [self.get_point()[0] + self.get_radius(), self.get_point()[1], self.get_height() + (Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS * 2)],
+            [self.get_point()[0] - self.get_radius(), self.get_point()[1], self.get_height() + (Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS * 2)]
```

Comment:
We might want to look at changing the additional height added since I added multiple points above the obstacle.

Reply:
done